### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by scjane**
The same as #2187 and #2188.

### CODE REVIEW

File: cs-engine/1.12/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ to update its RHEL kernel.
     $ sudo rpm --import "https://sks-keyservers.net/pks/lookup?op=get&search=0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e"
     ```
 
-    > **Note**: If the key server above does not respond, you can try one of these:
+    > **Note:** If the key server above does not respond, you can try one of these:
```
Comment (mstanleyjones) [inline cs-engine/1.12/index.md]: This one is incorrect. :)

File: cs-engine/1.12/index.md [hunk 2]
```diff
@@ -153,9 +153,10 @@ to update its RHEL kernel.
     $ curl -fsSL 'https://sks-keyservers.net/pks/lookup?op=get&search=0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e' | sudo apt-key add -
     ```
 
-    Note: if the key server above does not respond, you can try one of these:
-       - pgp.mit.edu
-       - keyserver.ubuntu.com
+    >**Note:** If the key server above does not respond, you can try one of these:
```
Comment (mstanleyjones) [inline cs-engine/1.12/index.md]: Again please don't include the colon in the bold text.

File: cs-engine/1.12/index.md [hunk 3]
```diff
@@ -233,9 +234,10 @@ to update its RHEL kernel.
     This adds the repository of the latest version of CS Docker Engine. You can
     customize the URL to install an older version.
 
-    Note: if the key server above does not respond, you can try one of these:
-       - pgp.mit.edu
-       - keyserver.ubuntu.com
+   **Note:** If the key server above does not respond, you can try one of these:
```
Comment (mstanleyjones) [inline cs-engine/1.12/index.md]: As above.

File: cs-engine/1.12/index.md (hunk 1 above)
Comment (scjane) [inline cs-engine/1.12/index.md]: In the documentation style guide, the "Note" style example given includes the colon in the bold text as follows. 
note
So, would you please update the style guide? Thanks.

Comment (mstanleyjones) [inline cs-engine/1.12/index.md]: Sure, see #2332 .

**Pull request merged**